tools/run-clang-tidy-in-ci.sh

```diff
@@ -28,4 +28,14 @@ if [[ ! -d build ]]; then
 fi
 
 # Run Clang-Tidy
-time python tools/clang_tidy.py -vp torch/csrc -d HEAD~1 "$@"
+time python tools/clang_tidy.py    \
+  --verbose                        \
+  --paths torch/csrc               \
+  --diff master                    \
+  -g"-torch/csrc/jit/init.cpp"     \
```

Comment:
Generally good to mention in comment why these are excluded.

Reply:
yes true, thanks

Reply:
Oh it's actually below. I'll put it above